server/src/gui.py

```diff
@@ -0,0 +1,150 @@
+import time
+import threading
+import random
+import queue
+from tkinter import *
+import tkinter.simpledialog
+from permissions import *
+import asyncio
+
+def server_start():
+    client.has_server_started = 1
+    admin_key = tkinter.simpledialog.askstring("Password", "Permissions must be set, please enter administrator password:", show='*')
+    check_permissions(admin_key)
+    client.running = 1
+    client.thread1.start()
+
+def quit_system():
+    if client.has_server_started:
+        admin_key = tkinter.simpledialog.askstring("Password", "Permissions must be returned to normal, please enter administrator password:", show='*')
+        restore_permissions(admin_key)
+        os.kill(os.getpid(),signal.SIGKILL)
+    else:
+        import sys
+        sys.exit(1)
+    
+
+logo='''
+                 ___________                 oo
+                / __/ __/ _ |             = "  $==
+               _\ \/ _// __ |             " o   $
+              /___/___/_/_|_|_  _  __    "      $
+                / /  /  _/ __ \/ |/ /   $      $$ 
+               / /___/ // /_/ /    /    o"     "$o 
+   $oo        /____/___/\____/_/|_/     $     "$$$
+   $"$oo  the friendly packet sniffer  "      "o$$ 
+  "$$o$"o"o        "ARP ARP"         o"      $"$$$
+     """o"$o"""""""" """ oo ooooooo"""     o""o$$$
+          "$$ o                            o""$$$
+    oooo$$$$$$$$oo"o o                 o "o "o ""oo
+   "$$$$$$"         ""$$o$o$ $ "ooo$o$o$$$$$$"$o$o$
+   "$""                 """"$$$$$o$$$$$$$$$     "$o$$o
+   "                           " "" "  "$$$$       " $o
+                                         "$$$         ""
+                                           ""$
+'''    
+
+class GuiPart:
+    def __init__(self, master, queue, endCommand):
+        self.queue = queue
```

Comment:
I can't tell where the queue is being used. Is it for future stuff or is it doing something here? If it's not being used, let's take it out + any functions that would process messages in it.